### DISCUSSION TITLE
Nemo model for brazillian portuguese

### DISCUSSION THREAD

**Discussion by danilokarkus** [NVIDIA-NeMo/NeMo#3784] [Q&A]
Hey everyone!
I am playing around with Nemo and got really excited qith the framework and all the built in functionalities!! the only thing is that i wanted to play around with audio data in Brazillian portuguese. Have anyone pre trained a nemo model for portuguese ? Thanks in advance!!!

**Comment by okuchaiev** [comment]
Unfortunately, at this moment we don't have any Portuguese models.
